### ISSUE TITLE
Namespaces can be used as types in various places

### ISSUE THREAD

**Issue by mor-n4** [eclipse/n4js#704]
```javascript
import * as N from "OtherModule";

// XPECT FIXME errors --> "must not use namespace as type" at "N"
let bad: N;

// XPECT FIXME errors --> "must not use namespace as type" at "N"
function worse(p: N): N {
	return undefined;
}
```

**Comment by mor-n4** [comment]
... and values:
```javascript
import * as N from "OtherModule";

// XPECT FIXME errors --> "must not use namespace as value" at "N"
N;
```

**Comment by yoosiba** [comment]
@mor-n4 after #222 is this bug still valid? Is it still in progress ?

**Comment by mor-n4** [comment]
Still valid, still in progress.

**Comment by mor-n4** [comment]
In plain Javascript, the namespace is actually an object that can be passed around, etc. So the above validations would introduce an unnecessary, undesired incompatibility with JS and are therefore rejected.

Closing this as *WON'T FIX*.
